content/oto-morph.md

```diff
@@ -0,0 +1,161 @@
+
+
+#### Our Mission
+With technologies such as translation systems, spell checkers, language-specific keyboards, and dictionary apps, it is possible for digitally disadvantaged languages to “digitally ascend,” empowering speakers to participate in global online society using their mother tongues [[4]](#4) [[11]](#11). This is particularly important for the 1.8 million Oto-Manguean speakers, 12\% of whom are monolingual; the development of language technologies for Oto-Manguean could choke back systematic forces that exhort linguistic assimilation [[2]](#2) [[12]](#12). Such forced linguistic assimilation has been known to cause irreparable damage to the groups affected [[13]](#13).
+
+Bird criticizes technologists who frame computational tasks for low-resource languages as “zero-resource scenarios,” ignoring the relevant knowledge of linguists and native speakers [[12]](#12). We believe that in the interest of language justice, solutions to the problem of automatic morphological inflection should be tailored to the language of interest, sharing cross-lingual information when necessary. For this reason, we adapt the ![equation](https://latex.codecogs.com/gif.latex?%5Cbg_white%20%5Ctexttt%7Bdeepspin-02-1%7D) system introduced by Peters and Martins [[7]](#7) to a much narrower domain--the Oto-Manguean language family--initializing language embeddings using prior knowledge of language similarity rather than doing so at random.
+
+### Methods
+
+#### Data
+
+We obtain our data directly from the SIGMORPHON 2020 Shared Task 0 repository [[1]](#1). It contains 10 Oto-Manguean languages from a wide range of branches (Figure 1). Thus, our data is representative of most of the language family. The train set size for each language ranges from 805 (Chichicapan Zapotec) to 22962 (Mezquital Otomi), with an average size of 7799.3 (Figure 2). 
+
+The task dataset comes from two resources. The data source for 9 of the languages comes from the Oto-Manguean inflectional class database [[14]](#14). The data for the remaining language, Eastern Highland Chatino, comes from a Chatino corpus released in 2020 [[15]](#15). 
+
+Each line of the data files is organized as follows:
+
+![equation](https://latex.codecogs.com/gif.latex?%5Cbg_white%20%5Cbegin%7Bquote%7D%20%5Ctexttt%7B%20lemma%20%5Cquad%20target%5C_form%20%5Cquad%20tags%20%7D%20%5Cend%7Bquote%7D)
```

Comment:
This is fine, but did you know that in [Pandoc markdown](https://pandoc.org/MANUAL.html#math-rendering-in-html), you can just write LaTeX inside `$` characters, and it will generate equations for you?

Reply:
good to know!